src/GZip.jl

```diff
@@ -414,7 +416,7 @@ function readall(s::GZipStream, bufsize::Int)
             if length(buf) > len
                 resize!(buf, len)
             end
-            return bytestring(buf)
+            return UTF8String(copy(buf))
```

Comment:
I think I misunderstood what I had read in Compat. Looking again, I think it should be `@compat String(copy(v))`?